test/e2e/util/util.go

```diff
@@ -378,10 +378,10 @@ func CreateManagedCluster(dynamicClient dynamic.Interface) (*unstructured.Unstru
 	}
 
 	// update fakeManagedCluster status with the new client
-	err = SetStatusType(fakeManagedCluster, clusterv1.ManagedClusterConditionJoined)
-	if err != nil {
-		return nil, err
-	}
+	// err = SetStatusType(fakeManagedCluster, clusterv1.ManagedClusterConditionJoined)
```

Comment:
is this intentional?

Reply:
yes. the status could not impact the e2e.
 it reports error  even if I update the lastTransitionTime with time.Now().Format("9090-09-09T51:951:08Z").
` ManagedCluster.cluster.open-cluster-management.io "test-automation-bxtgw" is invalid: status.conditions.lastTransitionTime: Invalid value: "9090-09-09T51:951:08Z": status.conditions.lastTransitionTime in body must be of type date-time: "9090-09-09T51:951:08Z"`

Reply:
so why we comment them instead removing them?

Reply:
I think the error could be fixed by the upgraded api when the condition PR is merged.

Reply:
gotcha